CHANGES.rst

```diff
@@ -26,6 +26,9 @@ Imviz
 
 - Improve performance by using FITS WCS for reference data layers when linked by WCS, rather than GWCS [#3483]
 
+- FootprintSelect tool now appears in the viewer toolbar only when
+  overlays are present, reducing toolbar clutter. [#3539]
```

Comment:
should we just append to the existing changelog entry since that was never released?